kci

```diff
@@ -13,28 +13,12 @@ features.  See the online documentation for more details:
 https://kernelci.org/docs/api/.
 """
 
-import json
 
-import click
-
-from kernelci.cli import Args, kci
+from kernelci.cli import kci
 from kernelci.cli import (  # pylint: disable=unused-import
     docker as kci_docker,
+    user as kci_user,
 )
```

Comment:
These are unused imports.

Reply:
Seems like it is required to register sub-commands. A comment about it would have been helpful.